.github/workflows/build.yml

```diff
@@ -33,62 +33,19 @@ jobs:
     name: Lint
     runs-on: ubuntu-22.04
     steps:
-      - name: Checkout repo
+      - name: Check out repo
         uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
 
-      - name: Set up dotnet
+      - name: Set up .NET
```

Comment:
⛏️  I'd keep this `dotnet` to keep it consistent with the CLI command run on line 43.